tests/integration/targets/ntnx_prism_v2/tasks/object_store.yml

```
@@ -0,0 +1,826 @@
+---
+- name: Start testing for some modules under Prism namespace (object store location)
```

```suggestion
- name: Start testing for modules for PC backup restore using objects store.
```

Add no log to all the tasks which involves creds usage